### DISCUSSION TITLE
Vivado "part xck26-sfvc784-2lv-c" not found

### DISCUSSION THREAD

**Discussion by liyuan-chang** [bol-edu/HLS-SOC-Discussions#61] [Q&A]
我在 Lab 2 要使用 KV260 時，Vivado 的 parts 選項找不到 xck26-sfvc784-2lv-c，但是在 boards 選項裡面卻有看到 Kria KV260，想請問這樣我是要直接選擇 Kria KV260 board file，還是要怎麼解決 parts 選項的問題呢？

![part_not_found](https://github.com/bol-edu/HLS-SOC-Discussions/assets/78146388/d4c3dfc7-3913-4f8f-a798-5bcc5d5df88b)

![board_found](https://github.com/bol-edu/HLS-SOC-Discussions/assets/78146388/bd945ea1-4e0d-4699-976a-0141962dd012)

**Comment by nthuyouwei** [comment]
直接選KV260就好了

liyuan-chang ***@***.***>於 2023年9月21日 週四，下午2:07寫道：
